allennlp/modules/augmented_lstm.py

```diff
@@ -0,0 +1,171 @@
+from typing import Optional, Tuple
+import torch
+from torch.autograd import Variable
+from torch.nn.utils.rnn import pad_packed_sequence, pack_padded_sequence, PackedSequence
+
+from allennlp.common.tensor import get_dropout_mask
+from allennlp.common.checks import ConfigurationError
+
+
+class AugmentedLstm(torch.nn.Module):
+    """
+    An LSTM with Recurrent Dropout and the option to use highway
+    connections between layers.
+
+    Parameters
+    ----------
+    input_size : int, required.
+        The dimension of the inputs to the LSTM.
+    output_size : int, required.
+        The dimension of the outputs of the LSTM.
+    direction: str, optional (default = "forward")
```

Comment:
Yeah - I just thought it wasn't necessarily obvious that `True => forward`, but if it's obvious to you then that's fine.